src/crypto/NoteEncryption.hpp

```diff
@@ -0,0 +1,89 @@
+/*
+See the Zcash protocol specification for more information.
+https://github.com/zcash/zips/blob/master/protocol/protocol.pdf
+
+Zcash allows users to send and recieve value using notes. In order
+to protect the privacy of the parties involved, this note is not
+public, and for verification purposes, is cryptographically
+commited. This commitment is public, and lives in the transaction.
+
+However, recipients must be able to obtain values of the note,
+including the value `v`, the trapdoor `r`, and a secret `rho`,
```

Comment:
"..., a secret `rho`, and the trapdoor `r`,"

(I always put them in that order to avoid confusion.)